### DISCUSSION TITLE
高德地图商用要收费5W了。。。

### DISCUSSION THREAD

**Discussion by qilovehua** [qiuxiang/react-native-amap3d#720] [General]
只能说这价格好高，app赚的钱还没那么多，有别的办法吗？

**Comment by qilovehua** [comment]
[https://www.zhihu.com/question/456440995](https://www.zhihu.com/question/456440995)

**Comment by qiuxiang** [comment]
那只能换了

**Comment by qiuxiang** [comment]
反正我个人是不喜欢高德地图，渲染帧率太低。所以我后来又做了百度地图封装，用的百度地图，虽然现在因为项目不维护了，组件也不维护了。

我在 flutter 一开始用的是腾讯地图和百度地图，但后来发现 flutter 的原生 view 封装对性能有损失，后来干脆连百度地图和腾讯地图都不用了，就用瓦片地图，瓦片地图在 flutter 下的实现就非常丝滑，虽然某些方面体验和效果是不如 3d 引擎渲染出来的，但好在可以随意换贴图源，再不济可以换成天地图的贴图。

**Reply by qilovehua** [reply]
看了下react-native-mapbox-gl/maps国内不能直接用，我先去了解下瓦片地图。
谢谢大佬！

**Comment by qilovehua** [comment]
您的（应用截图）在未经授权的情况下，使用第三方（高德）产品素材，若已获得相关授权，请在特殊资质栏提供相关授权证明，若没有授权证明，请删除（商标、品牌标识）后重新提交。

小米审核失败，高德这是要放大招了。主要是用户量太小了，一年5W赚不到那么多啊。

**Reply by qilovehua** [reply]
搞错了，是应用截图上不能有高德logo
